validator/src/main.rs

```diff
@@ -2114,6 +2114,12 @@ pub fn main() {
         .collect::<HashSet<_>>()
         .into_iter()
         .collect::<Vec<_>>();
+    for addr in &entrypoint_addrs {
+        if !socket_addr_space.check(addr) {
```

Comment:
hehe. we need to rename this method so i don't have to look it up to remember what it's "checking"

Reply:
yeah, I guess in retrospect `socket_addr_space.contains(addr)` was a better name.